modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -241,6 +251,20 @@ struct IEUnit {
         if (params.num_out == 1u && params.output_names.empty()) {
             params.output_names = { outputs.begin()->first };
         }
+        // Reshape for configured map with layers' names and their new shapes
+        if (!params.reshape_in_shapes.empty()) {
+            GAPI_Assert(params.reshape_in_shapes.size() <= params.num_in);
+            // Checks identity of input shape size and current CNN shape
+            // Assumes that layout of input shape is correct
+            for (const auto& el : params.reshape_in_shapes) {
+                const auto idx = getIdxByName(params.input_names, el.first);
```

Comment:
Are u sure that this function is really necessary ? After `std::find` you have already have `iterator` which can be dereferenced to obtain `string`.

Reply:
Removed.